### PR TITLE
master-lps-114308

### PR DESCRIPTION
**Pull request by Jeremy-Ch**
https://issues.liferay.com/browse/LPS-114308

### CODE REVIEW

**Comment by Jeremy-Ch** [comment]
ci:test:sf

**Comment by Jeremy-Ch** [comment]
ci:test:sf

**Comment by austinchiang** [comment]
**Reviewing...** :hourglass_flowing_sand: :sunglasses: :headphones:

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/Translation.path [hunk 1]
```diff
@@ -44,6 +44,11 @@
 	<td>//div[contains(@class,'localized')]//button[contains(@class,'localized') and contains(@id,'description')][*[normalize-space(text())='${key_locale}']]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>DROPDOWN_MENU_ITEM</td>
```
Comment (austinchiang) [inline portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/Translation.path:53]: Is this necessary? Can we use ``MenuItem#ANY_MENU_ITEM``?

Comment (austinchiang) [reply portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/Translation.path:53]: Or ``MenuItem#DROPDOWN_MENU_ITEM``? We should try to reuse when we can

File: modules/apps/translation/translation-test/src/testFunctional/macros/Translations.macro [hunk 2]
```diff
@@ -283,7 +283,7 @@ definition {
 				key_fieldValue = "${webContentDecimal}",
 				locator1 = "Translation#TARGET_LANGUAGE_FIELD");
 		}
-	}	
+	}
```
Comment (austinchiang) [inline modules/apps/translation/translation-test/src/testFunctional/macros/Translations.macro:286]: Per our conventions please do not SF on top of your own commits

File: modules/apps/translation/translation-test/src/testFunctional/macros/Translations.macro [hunk 3]
```diff
@@ -13,6 +13,38 @@ definition {
 		}
 	}
 
+	macro changeOriginalLocale {
```
Comment (austinchiang) [inline modules/apps/translation/translation-test/src/testFunctional/macros/Translations.macro:16]: I would change all references of `Original` to `Base`

**Comment by austinchiang** [comment]
@Jeremy-Ch  Please make the necessary changes and resend. Thanks!

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/Translation.path (hunk 1 above)
Comment (Jeremy-Ch) [reply portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/Translation.path:53]: I have tried these two paths, they are not working for this test.

File: modules/apps/translation/translation-test/src/testFunctional/macros/Translations.macro (hunk 2 above)
Comment (Jeremy-Ch) [reply modules/apps/translation/translation-test/src/testFunctional/macros/Translations.macro:286]: Oh, sorry, I didn't know this before

**Pull request closed without merging**